src/controllers/auth/login.js

```diff
@@ -24,9 +24,10 @@ define(["apphost", "appSettings", "dom", "connectionManager", "loading", "layout
             page.querySelector("#txtManualPassword").value = "";
             loading.hide();
 
-            if (response.status === 401) {
+            if (response.status === 401 || response.status === 403) {
                 require(["toast"], function (toast) {
-                    toast(Globalize.translate("MessageInvalidUser"));
+                    var messageKey = response.status === 401 ? "MessageInvalidUser" : "MessageUnauthorizedUser";
```

Comment:
Feel free to use `let` or `const` in modules, everything is passed through Babel (not only what is explicitly set to be an ES6 module).

Reply:
Done 👍 So nice to be able to use modern features :D

Reply:
We have polyfill linting too, now, so you can use new APIs and it'll warn you if stuff needs a polyfill :)